crypto/err/err.c

```diff
@@ -654,18 +613,9 @@ const char *ERR_lib_error_string(unsigned long e)
 
 const char *ERR_func_error_string(unsigned long e)
 {
-    ERR_STRING_DATA d, *p;
-    unsigned long l, f;
-
-    if (!RUN_ONCE(&err_string_init, do_err_strings_init)) {
+    if (!RUN_ONCE(&err_string_init, do_err_strings_init))
         return NULL;
-    }
-
-    l = ERR_GET_LIB(e);
-    f = ERR_GET_FUNC(e);
-    d.error = ERR_PACK(l, f, 0);
-    p = int_err_get_item(&d);
-    return ((p == NULL) ? NULL : p->string);
+    return ERR_GET_LIB(e) == ERR_LIB_SYS ? "system library" : NULL;
```

Comment:
No.  Just return NULL.  Having the function indication saying "system library" is redundant and frankly confusing.  *The library indication already says this*

Reply:
Changes being made.  I think keeping ERR_R_DISABLED is a very bad idea, turning compile-time error into possibly run-time.  There's a reason why it was never used :)  But okay it's back.

Reply:
It would actually be perfectly suitable in places like this:

https://github.com/openssl/openssl/blob/7a228c391e0a35e1dc1223e3af3371968376857b/crypto/bio/bss_file.c#L420-L423

You know, in place where we *do* signal an error of that type in run-time.

Reply:
Shrug.  It's not used now.  And I believe a more proper fix is to, ya know, not include the function (unless needed for backward compatibilty).  But I am being needlessly pedantic; pax.